src/server/actions/ai.ts

```diff
@@ -133,6 +133,9 @@ export const getAgentKit = async ({
   const agent = new SolanaAgentKit(walletAdapter, RPC_URL, {
     OPENAI_API_KEY: openaiKey,
     HELIUS_API_KEY: process.env.HELIUS_API_KEY!,
+    ...(process.env.HELIUS_RPC_URL && {
+      HELIUS_RPC_URL: process.env.HELIUS_RPC_URL,
+    }),
```

Comment:
SolanaAgentKit hardcodes the helius rpc url to be mainnet. This makes it impossible to test with devnet. The only way I could have made this work was to 

- fork the SolanaAgentKit
- added a new HELIUS_RPC_URL which allows me to override the mainnet default url with a devnet one 
- install the forked SolanaAgentKit in package.json (to ensure nothing breaks from the existing implementation, the fork SolanaAgentKit is rebase from github:slimeonmyhead/solana-agent-kit#c571e8b59ed7d399a705578af2d58c95f7e9e826 (@slimeonmyhead forked repo and the commit c571e8b59ed7d399a705578af2d58c95f7e9e826)